docs/sources/source/nlp/joint_intent_slot_filling.rst

```diff
@@ -128,17 +126,14 @@ Next, we define all Neural Modules participating in our joint intent slot fillin
                 num_samples=num_samples,
                 shuffle=shuffle,
                 batch_size=batch_size,
-                num_workers=0,
-                local_rank=local_rank,
                 ignore_extra_tokens=args.ignore_extra_tokens,
-                ignore_start_end=args.ignore_start_end
-                )
+                ignore_start_end=args.ignore_start_end,
+            )
 
-            ids, type_ids, input_mask, loss_mask, \
-                subtokens_mask, intents, slots = data_layer()
+            (ids, type_ids, input_mask, loss_mask, subtokens_mask, intents, slots) = data_layer()
```

Comment:
please use the Toms port refactoring, which is input_data = data_layer(), input_data.ids, input_data.type_ids etc

Reply:
Done.